collections/sentinel-1-grd.yaml

```diff
@@ -110,14 +110,14 @@ CubeDimensions:
     type: spatial
     axis: y
     extent:
-      - -85
-      - 85
+      - -90
+      - 90
   t:
     type: temporal
     extent:
-      - '2014-12-07T04:14:15Z'
+      - '2014-10-03T04:14:15Z'
       - null
-    step: P5D
+    step: P6D
```

Comment:
Where did this come from? It seem like some EWs have daily revisit. eg. https://sentinelshare.page.link/nRGJ

Reply:
It's from [here](https://sentinels.copernicus.eu/web/sentinel/user-guides/sentinel-1-sar/revisit-and-coverage) but I should have read the whole thing. Set to "null" as it depends on the location = irregularly spaced steps.